### PR TITLE
Add pre-application journeys with stubbed content

### PR DESCRIPTION
**Pull request by ptrelease**
Initial changes to introduce pre-applications such as project enquiries and expressions of interest to funding-frontend 

Added new models, views, controllers and migrations to accomplish this

### CODE REVIEW

File: app/controllers/pre_application/expression_of_interest/feasibility_or_options_work_controller.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Controller for the Pre-application 'Start' page
```
Comment (ptrelease) [inline app/controllers/pre_application/expression_of_interest/feasibility_or_options_work_controller.rb:1]: Change this

File: app/controllers/pre_application/expression_of_interest/likely_submission_description_controller.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    @pre_application.pa_expression_of_interest.update(pa_expression_of_interest_params)
+
+    if @pre_application.pa_expression_of_interest.valid?
+
+      logger.info 'Finished updating likely_submission_description for pa_expression_of_interest ID: ' \
+                  "#{@pre_application.pa_expression_of_interest.id}"
+      
+      the_json = @pre_application.to_salesforce_json
+
+      logger.info(the_json)
```
Comment (ptrelease) [inline app/controllers/pre_application/expression_of_interest/likely_submission_description_controller.rb:24]: Not sure we should be logging,  consider how long logs held for.  May be better to store on DB, so we can easily clear later as batch or scheduled job.  Same applies to PEF.

File: app/controllers/pre_application/expression_of_interest/submitted_controller.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Controller for the Pre-application 'Start' page
+class PreApplication::ExpressionOfInterest::SubmittedController < ApplicationController
+    include PreApplicationContext
+
+end
```
Comment (ptrelease) [inline app/controllers/pre_application/expression_of_interest/submitted_controller.rb:5]: add newline

**Comment by ptrelease** [comment]
@stuartmccoll Looks good to me - Can't approve myself but happy when you are.

**Review by stuartmccoll — APPROVED** [review]

**Pull request merged**